bpfd/src/serve.rs

```diff
@@ -98,6 +97,12 @@ pub async fn serve(
         }
     };
     if csi_support {
+        const VERSION: &str = env!("CARGO_PKG_VERSION");
+        let node_id = std::env::var("KUBE_NODE_NAME")
+            .expect("cannot start bpfd csi driver KUBE_NODE_NAME not set");
+        let storage_manager =
+            StorageManager::new(DRIVER_NAME.to_string(), node_id, VERSION.to_string());
```

Comment:
This should all just be computed in `StorageManager::new()`.
No need to pass it into the constructor.
```suggestion
            StorageManager::new();
```